README.md

```diff
@@ -46,11 +46,21 @@ requests, code review feedback, and also pull requests.
 
 ## Supported Java Versions
 
-We currently support Java 7 and higher. However, Java 7 support is deprecated.
-We strongly encourage you to use Java 8 or higher as we will drop support for
-Java 7 in the next major version. Firebase Admin Java SDK also runs on [Google App
+We currently support Java 8 and higher. Firebase Admin Java SDK also runs on [Google App
```

Comment:
Nit: use "The Firebase Admin Java SDK" like below.

Reply:
Thank you! Fixed.